T/Torch/build_tarballs.jl

```diff
@@ -0,0 +1,61 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Torch"
+version = v"0.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/dhairyagandhi96/Torch.jl.git", "985c4280c17fd9b0af978760be66761456a0113b"),
+    FileSource("https://download.pytorch.org/libtorch/cu101/libtorch-cxx11-abi-shared-with-deps-1.4.0.zip", "f214bfde532877aa5d4e0803e51a28fa8edd97b6a44b6615f75a70352b6b542e"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+mv libtorch/ $WORKSPACE/destdir/
+cd /usr/local/
+mkdir cuda
+cd cuda
+mkdir lib64
+cd lib64/
+ln -s $WORKSPACE/artifacts/ce02776dfe49896031bc14077df1e002f26396d4/lib64/libcudart.so libcudart.so
+ln -s $WORKSPACE/artifacts/ce02776dfe49896031bc14077df1e002f26396d4/lib64/libnvToolsExt.so libnvToolsExt.so
+cd $WORKSPACE/destdir/
+mkdir cudnn && cd cudnn
+wget https://github.com/JuliaGPU/CUDABuilder/releases/download/v0.3.0/CUDNN+CUDA10.1.v7.6.5.x86_64-linux-gnu.tar.gz
+tar -xvf CUDNN+CUDA10.1.v7.6.5.x86_64-linux-gnu.tar.gz 
+rm CUDNN+CUDA10.1.v7.6.5.x86_64-linux-gnu.tar.gz 
```

Comment:
Why not adding this a source instead?  Isn't this available as a JLL package?

Reply:
I don't think it's in Yggdrasil, and adding it as a source also unpacked the tarball in the wizard build environment. So I wanted to avoid it. What would the proper way be?

Reply:
Maybe you want to use the `unpack_target` keyword argument of [`ArchiveSource`](https://juliapackaging.github.io/BinaryBuilder.jl/dev/reference/#BinaryBuilder.ArchiveSource)?

Reply:
I see, I think using the wizard kind of colored my view of how it would actually be initialized

Reply:
Bump :slightly_smiling_face:

Reply:
Thanks! Is this somewhat closer to what you had in mind? I also think it better to move it out of destdir, otherwise it will ship cudnn, which is large as well